src/common/options/rbd-mirror.yaml.in

```diff
@@ -2,6 +2,13 @@
 ---
 
 options:
+- name: rbd_mirror_remote_osd_op_timeout
+  type: secs
+  level: advanced
+  desc: timeout for OSD operations against the remote cluster (0 is unlimited)
+  default: 30
```

Comment:
On clusters with synthetically injected 120ms per-packet latency, we have seen some OSD ops (4M reads generated when resyncing an image from scratch in particular) spend over a minute in the messenger, so I think we need to be much more conservative with the default.

Perhaps 5 minutes to match `client_mount_timeout`?